laravel/the-basics/databases.html.md

```diff
@@ -0,0 +1,198 @@
+
+First, open your MySQL application&#39;s fly.toml and take note of the following:
+
+```toml
+app = "<mysql-app-name>"
+
+[env]
+  MYSQL_DATABASE = "<database-name>"
+  MYSQL_USER =  "<database-user>"
+```
+
+Then use `flyctl proxy`  to tunnel to your MySQL application:
+
+```cmd
+flyctl proxy 3306 -a <mysql-app-name>
+```
+
+Finally, update your local .env file and update with the values from your MySQL `fly.toml` file:
+
+```env
+DB_CONNECTION=mysql
+DB_HOST=127.0.0.1
+DB_PORT=3306
+DB_DATABASE=<MYSQL_DATABASE>
+DB_USERNAME=<MYSQL_USER>
+DB_PASSWORD=<MYSQL_PASSWORD>
+```
+
+
+
+## _MySQL on PlanetScale_
+
+PlanetScale is a serverless, MySQL database-compatible platform built for horizontally scaling data.
+
+In this setup, you&#39;ll find a basic setup of PlanetScale and Fly.io connection. However, for a multi-region level up, checkout our [Multi-Region Laravel with PlanetScale](https://fly.io/laravel-bytes/multi-region-laravel-with-planetscale/) article.
+
+Before you can connect your Laravel [application in Fly.io to a PlanetScale instance](https://fly.io/docs/app-guides/planetscale/), you&#39;ll have  to tread several steps first:
+
+1. Setup a  PlanetScale database instance
+1. Get connection information from PlanetScale instance
+1. Connect from Laravel application in Fly.io
+
+
+
+### Setup a PlanetScale database instance
+
+You can check out their quick-start guide [here](https://planetscale.com/docs/tutorials/planetscale-quick-start-guide), or follow along:
+<ol>
+<li>First make sure you have an account, and [login](https://auth.planetscale.com/sign-in?return_to=%2F)</li>
+<li>You&#39;ll be redirected to your organization dashboard, where you&#39;ll be able to find a `"create a database"` link.</li>
+<li>Click on the create link, and hopefully you&#39;ll get  a prompt to add in your preferred database name and region</li>
+</ol>
+
+![PlanetScale initialize database dialog](/docs/images/planetscale_laravel_new_db_dialog.png)
+
+### Get connection information from PlanetScale instance
+
+Once initialized, your database dashboard should have metrics and options like so:
+
+![PlanetScale initialize database dashboard](/docs/images/planetscale_laravel_new_database.png)
+<ol>
+<li>Click on the Connect button at the top right, this should provide a box of information for connecting with your PlanetScale database.</li>
+<li>First though, make sure to add a password,  by clicking on the &quot;New Password&quot; button at the upper right corner. This should show you a new password afterwards.</li>
+<li>Next, select &quot;Laravel&quot; in the list labeled &quot;Connect with&quot;</li>
+</ol>
+
+![PlanetScale Laravel connection string](/docs/images/planetscale_laravel_connection_string.png)
+Take note of the connection string provided and let&#39;s move on!
+
+### Connect from a Laravel application in Fly.io
+
+Update your Laravel application&#39;s fly.toml&#39;s `[env]` with details from the PlanetScale connection string 
```

Comment:
fly.toml -> `fly.toml` (to be consistent)

Reply:
nice find, fixing!